src/features/click-to-load/assets/ctl-placeholder-block.css

```diff
@@ -0,0 +1,329 @@
+:host,
+* {
+    font-family: DuckDuckGoPrivacyEssentials, system, -apple-system, system-ui, BlinkMacSystemFont, 'Segoe UI', Roboto,
+        Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol';
+    box-sizing: border-box;
+    font-weight: normal;
+    font-style: normal;
+    margin: 0;
+    padding: 0;
+    text-align: left;
+}
+
+:host,
+.DuckDuckGoSocialContainer {
+    display: inline-block;
+    border: 0;
+    padding: 0;
+    margin: auto;
+    inset: initial;
+    max-width: 600px;
+    min-height: 180px;
+}
+
+/* SHARED STYLES */
+/* Button */
+.DuckDuckGoButton {
+    border-radius: 8px;
+    padding: 11px 22px;
+    margin: 0px auto;
+    border-color: #3969ef;
+    border: none;
+    height: 36px;
+    font-size: 14px;
+
+    position: relative;
+    cursor: pointer;
+    box-shadow: none;
+    z-index: 2147483646;
+}
+.DuckDuckGoButton > div {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    border: none;
+    padding: 0;
+    margin: 0;
+}
+.DuckDuckGoButton.tertiary {
+    color: rgba(0, 0, 0, 0.84);
+    background-color: transparent;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    padding: 0px 16px;
+    border: 1px solid rgba(0, 0, 0, 0.12);
```

Comment:
Let's move all values that match design tokens in Figma into a `variables.css` file as CSS custom properties. Then you can re-use that file in this + any new components whilst sharing the values.